src/main/java/com/vmware/vipclient/i18n/base/cache/FormatCacheItem.java

```diff
@@ -4,28 +4,58 @@
  */
 package com.vmware.vipclient.i18n.base.cache;
 
-import java.util.HashMap;
-import java.util.Map;
+import com.vmware.vipclient.i18n.VIPCfg;
 
 public class FormatCacheItem implements CacheItem {
+
+	private String etag;
+	private long timestamp;
+	private Long maxAgeMillis = 86400000l;
+
 	public FormatCacheItem() {
-		
+
+	}
+
+	public String getEtag() {
+		return etag;
+	}
+
+	protected void setEtag(String etag) {
```

Comment:
Since these properties are private, can't be accessed by sub-classes, so I have to make the setter methods as protected or make the properties themselves as protect to enable them be accessed by sub-classes.

And I remember you said before it's a good practice to change the property's value with its' setter method instead of change the value directly, so I make the setter methods as protected.

So the setter methods can only be called in the sub-classes and in the package, can't be called by other classes outside.